docs/user/alerting/defining-alerts.asciidoc

```diff
@@ -74,6 +74,21 @@ image::images/alert-flyout-add-action.png[You can add multiple actions on an ale
 Actions are not required on alerts. In some cases you may want to run an alert without actions first to understand its behavior, and configure actions later.
 ==============================================
 
+[float]
+=== Global actions configuration
+Some actions configuration options apply to all actions.
+If you are using an *on-premises* Elastic Stack deployment, you can set these in the kibana.yml file.
+If you are using a cloud deployment, you can set these via the console.
+
+Here's a list of the available global configuration options and an explanation of what each one does:
+
+* `xpack.actions.allowedHosts`: Set to an array of host names which actions such as email, slack, pagerduty, and webhook can connect to. An element of * indicates any host can be connected to. An empty array indicates no hosts can be connected to. Default: [ {asterisk} ]
```

Comment:
Should Slack, PagerDuty, and Webhook be capitalize?

Suggestion:

Set to an array of host names **that** actions such as email, slack, pagerduty, and webhook can connect to. An * indicates any host, and an empty array indicates no hosts.

Reply:
Slack and PagerDuty, yes, webhook is not a product but a generic term, so no.